rwengine/src/audio/SfxParameters.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef _RWENGINE_SFX_PARAMETERS_HPP_
+#define _RWENGINE_SFX_PARAMETERS_HPP_
+
+#include <cstddef>
+
+/// Nr of existing sfx sounds
+constexpr size_t kNrOfAllSfx = 3032;
```

Comment:
Like @madebr noted previously this looks to depend of the game and this is correct only for GTA III?

Reply:
Yeah.

Reply:
I think Todo might be enough at this point (as also the SfxData array in SfxParameters.cpp is valid only for GTAIII?)